bin/zkCleanup.sh

```diff
@@ -31,27 +31,44 @@
 # use POSIX interface, symlink is followed automatically
 ZOOBIN="${BASH_SOURCE-$0}"
 ZOOBIN="$(dirname "$ZOOBIN")"
-ZOOBINDIR="$(cd "$ZOOBIN" && pwd)"
+ZOOBINDIR="$(cd "$ZOOBIN" || exit && pwd)"
 
 if [[ -e "$ZOOBIN/../libexec/zkEnv.sh" ]]; then
-  # shellcheck source=bin/zkEnv.sh
```

Comment:
Why remove the shellcheck hints? Those help shellcheck find the scripts in the source code when doing validation.